### DISCUSSION TITLE
brew & BIND named on old ElCapitan

### DISCUSSION THREAD

**Discussion by macuserguru** [Homebrew/discussions#3068] [Getting started]
How I could install brew & BIND named on old ElCapitan?
All previous attempts failed. 
I want to test this before I install them on my new M1 Mac (Monterey).

**Comment by SMillerDev** [comment]
No, El Capitan is no longer supported

**Reply by macuserguru** [reply]
I just want to know how the deamon of BIND named runs with homebrew installed. Would have liked to try this on another Mac.
a) Under which user account? MacPorts creates a separate user for this - and several others anyway.
b) If under root, then in a sandbox?
c) Where can I find detailed instructions on how to configure BIND named under Homebrew and what security settings to use with Monterey.
PS: I could not get the named Deamon to work with Mac Ports. Some security setting prevented that. Since I want to try it with Homebrew - but before I have to inform myself.

I work with a M1 MacMini under Monterey.

**Reply by SMillerDev** [reply]
> Under which user account?

Homebrew only installs software, it doesn't create accounts. If you use brew services it'll run as the current user.

> Where can I find detailed instructions on how to configure BIND named under Homebrew and what security settings to use with Monterey.

We only distribute software, we don't document it. If BIND developers didn't document this I'm not sure where to find the documentation either.

**Comment by macuserguru** [comment]
This is a different philosophy than MacPorts.
This also explains why I would like to try this on another Mac. Unfortunately, your ElCapitan no longer supports it.
But the old PowerPC CPU Macs are still supported.

I don't want to tinker on the new Mac.

**Reply by SMillerDev** [reply]
Anything older than Catalina is unsupported. Homebrew (and Apple) only support 3 macOS versions at a time.

**Comment by macuserguru** [comment]
There are many older Mac in use - you are cutting their software needs.
I bought my two old macs in 2009 and they are still working - unfortunately without security updates.

**Reply by SMillerDev** [reply]
If homebrew had unlimited volunteers and unlimited CI machines that might have been possible. But with the amount of maintainers matching apple is the best we're going to do.

I'd much rather have homebrew working well on 3 versions than barely on all of them.

Macports supports older macOS versions though.

**Reply by macuserguru** [reply]
If I understand right, homebrew supports only Monterey, Big Sur, Catalina?
Whats with Mojave, the last 32Bit ready macOS?
Here I had a chance to upgrade this with an hack.

**Reply by SMillerDev** [reply]
> If I understand right, homebrew supports only Monterey, Big Sur, Catalina?

At this moment, yes

> Whats with Mojave, the last 32Bit ready macOS?

It's not supported

**Comment by macuserguru** [comment]
Now I found this two hints:
https://gist.github.com/mpaskalev/207ac091495ef648e269?permalink_comment_id=2965467
https://gist.github.com/Ikkyusan1/4e79e1ac844a2073223c2c4690931e77
